Pipfile

```diff
@@ -10,6 +10,7 @@ pytest-asyncio = "*"
 moto = "*"
```

Comment:
Need to have package version here to avoid compability issue since this is a open package installer.